tests/test_add.py

```diff
@@ -50,3 +54,28 @@ def test_get(self):
         rsp.render()
         body = rsp.content.decode('utf-8')
         self.assertIn('method="POST"', body)
+
+    def test_setting_form_class(self):
+        class DummyForm(forms.Form):
+            pass
+
+        glob = {}
```

Comment:
When I think 'glob', I think [this](https://docs.python.org/2/library/glob.html). Maybe `global_dictionary` or something to make it super obvious?